src/adcm_client/objects.py

```diff
@@ -1767,10 +1767,19 @@ def group_list(self) -> "GroupList":
 
     def change_password(self, password: str) -> None:
         """Changing user password"""
+        need_auth = False
+        if adcm_version.compare_adcm_versions(self.adcm_version, "2022.01.31.00") >= 0:
```

Comment:
I suggest using the actual version of ADCM: `2022.02.01.06`

Reply:
Good point. Also put comment above.